CMakeLists.txt

```diff
@@ -65,14 +66,9 @@ elseif (NOT MSVC)
   message(FATAL_ERROR "Unknown compiler")
 endif ()
 
-if (CMAKE_VERSION VERSION_LESS "3.1")
-  if (GCC)
-    set(CMAKE_C_FLAGS "-std=c99 ${CMAKE_C_FLAGS}")
-  endif ()
-else ()
-  set(CMAKE_C_STANDARD 99)
-endif ()
+set(CMAKE_C_STANDARD 99)
 
+option(BUILD_SHARED_LIBS "Build using shared libraries" ON)
```

Comment:
Is this a builtin option? I didn't catch any usage over the diff

Reply:
Yes! See the change on L88, this will inherit to added targets.

Reply:
https://github.com/uael/sv/pull/57/files#diff-1e7de1ae2d059d21e1dd75d5812d5a34b0222cef273b7c3a2af62eb747f9d20aL95

Reply:
https://cmake.org/cmake/help/latest/variable/BUILD_SHARED_LIBS.html